DZ Robot/src/by/teachmeskills/robot/Run.java

```diff
@@ -0,0 +1,53 @@
+package by.teachmeskills.robot;
+
+import by.teachmeskills.robot.hands.SamsungHand;
+import by.teachmeskills.robot.hands.SonyHand;
+import by.teachmeskills.robot.hands.ToshibaHand;
+import by.teachmeskills.robot.heads.SamsungHead;
+import by.teachmeskills.robot.heads.SonyHead;
+import by.teachmeskills.robot.heads.ToshibaHead;
+import by.teachmeskills.robot.legs.SonyLeg;
+import by.teachmeskills.robot.legs.ToshibaLeg;
+
+public class Run {
+    private int price;
+
+    public static void main(String[] args) {
+        /**
+         Создать по 3 реализации(Sony, Toshiba, Samsung) каждой запчасти(IHead, IHand, ILeg)
+         Класс SonyHead является примером реализацией головы от Sony.
+         Создайте 3 робота с разными комплектующими.
+         Например у робота голова и нога от Sony а, рука от Samsung.
+         У всех роботов вызовите метод action.
+         Среди 3-х роботов найдите самого дорогого.
+         */
+        SamsungHand samsungHand = new SamsungHand();
+        SamsungHead samsungHead = new SamsungHead();
+        SonyLeg sonyLeg = new SonyLeg();
+        SonyHead sonyHead = new SonyHead();
+        SonyHand sonyHand = new SonyHand();
+        ToshibaHead toshibaHead = new ToshibaHead();
+        ToshibaHand toshibaHand = new ToshibaHand();
+        ToshibaLeg toshibaLeg = new ToshibaLeg();
+
+        Robot robot1 = new Robot(samsungHead, samsungHand, sonyLeg);
+        Robot robot2 = new Robot(toshibaHead, toshibaHand, sonyLeg);
+        Robot robot3 = new Robot(sonyHead, sonyHand, toshibaLeg);
+        System.out.println("Первый робот:");
+        robot1.action();
+        System.out.println("Второй робот:");
+        robot2.action();
+        System.out.println("Третий робот:");
+        robot3.action();
+        if (robot1.getPrice() > robot2.getPrice() && robot1.getPrice() > robot3.getPrice()) {
+            System.out.println("Самый дорогой робот #1");
+        }
+        if (robot2.getPrice() > robot1.getPrice() && robot2.getPrice() > robot3.getPrice()) {
+            System.out.println("Самый дорогой робот #2");
+        }
+        if (robot3.getPrice() > robot1.getPrice() && robot3.getPrice() > robot2.getPrice()) {
+            System.out.println("Самый дорогой робот #3");
+        }
```

Comment:
and what if you have more than 3 robots? move this logic to a separate class and make it work with array